### PR TITLE
Complete JavaScript operators and expressions exercise.

### PR DESCRIPTION
**Pull request by dennis-pnguyen**
javascript-operators-and-expressions

### CODE REVIEW

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const final = 200;
+
+const grade = (quizzes + midterm + final) / 3;
+
+console.log('value of grade is:', grade);
+console.log('typeof grade is:', typeof grade);
+
+const firstName = "Dennis";
+const lastName = "Nguyen";
+let fullName = firstName + ' ' + lastName;
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js]: This should be `const`. `fullName` never gets reassigned, so it's value will always stay the same, so it should be declared using `const`

File: javascript-operators-and-expressions/main.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+console.log('typeof isAcidic is:', typeof isAcidic);
+
+const headCount = 420;
+const isSparta = headCount === 300;
+
+console.log('value of isSparta is:', isSparta);
+console.log('typeof isSparta is:', typeof isSparta);
+
+let motto = fullName;
+motto = fullName += ' is the GOAT!';
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js]: `+=` replaces `=` in this case. The `+=` operator concatenates or adds values, and then reassigns it to the variable. So this should just be `motto += ' is the GOAT!'`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good! Just a couple small tweaks here

File: javascript-operators-and-expressions/main.js (hunk 2 above)
Comment (dennis-pnguyen) [reply javascript-operators-and-expressions/main.js]: Screenshot 2023-07-10 152924
Screenshot 2023-07-10 152933

That's what I was missing, thanks Shawn!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**